src/app/routes/loan/loan.module.ts

```diff
@@ -4,8 +4,8 @@ import { LoanComponent } from '../loan/loan.component';
 // Routing
 import { SharedModule } from '$shared';
 import { CardModule } from 'primeng/card';
-import { ProgressSpinnerModule } from 'primeng/progressspinner';
 import { HelpPanelComponent, IconsComponent, MasterPageModule } from '../../components';
+import { LoadingSpinnerComponent } from '../../components/loading-spinner/loading-spinner.component';
```

Comment:
I would recommend using the TS path for imports from the global components directory. Simplifies things and makes it easier to break things out if we hoist this component to a global repo somewhere. I think this path is already setup, if not feel free to add.

```
import { LoadingSpinnerComponent } from '$components';
```

Reply:
I like using the dollar sign to indicate internal paths as to note confuse them with npm libs